packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -162,6 +164,34 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
       config.denominatorPriceFeed && (await _createMedianizerPriceFeed(config.denominatorPriceFeed));
 
     return new BasketSpreadPriceFeed(web3, logger, baselinePriceFeeds, experimentalPriceFeeds, denominatorPriceFeed);
+  } else if (config.type === "test") {
+    const requiredFields = ["currentPrice", "historicalPrice"];
```

Comment:
Yes in general I agree with your statement. But these fields are required for the `test` type of pricefeed because they are required inputs to construct a new `PriceFeedMockScaled`